### PR TITLE
[BottomSheet] Make `sheetView` a property.

### PR DESCRIPTION
**Pull request by romoore**
To make testing a bit easier, sheetView can become a property declared
in a class extension.

Part of #4945

### CODE REVIEW

**Review by yarneo — APPROVED** [review]
so much cleaner!

**Review by codeman7 — APPROVED** [review]

File: components/BottomSheet/src/MDCBottomSheetPresentationController.m [hunk 1]
```diff
@@ -161,8 +164,8 @@ - (void)viewWillTransitionToSize:(CGSize)size
 
   [coordinator animateAlongsideTransition:
       ^(__unused id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
-        self->_sheetView.frame = [self frameOfPresentedViewInContainerView];
-        [self->_sheetView layoutIfNeeded];
+        self.sheetView.frame = [self frameOfPresentedViewInContainerView];
+        [self.sheetView layoutIfNeeded];
         [self updatePreferredSheetHeight];
       }                        completion:nil];
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/BottomSheet/src/MDCBottomSheetPresentationController.m]: Done

File: components/BottomSheet/src/MDCBottomSheetPresentationController.m [hunk 2]
```diff
@@ -96,14 +99,14 @@ - (void)presentationTransitionWillBegin {
     scrollView = MDCBottomSheetGetPrimaryScrollView(self.presentedViewController);
   }
   CGRect sheetFrame = [self frameOfPresentedViewInContainerView];
-  _sheetView = [[MDCSheetContainerView alloc] initWithFrame:sheetFrame
+  self.sheetView = [[MDCSheetContainerView alloc] initWithFrame:sheetFrame
                                                 contentView:self.presentedViewController.view
                                                  scrollView:scrollView];
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/BottomSheet/src/MDCBottomSheetPresentationController.m]: Done

File: components/BottomSheet/src/MDCBottomSheetPresentationController.m [hunk 3]
```diff
@@ -161,8 +164,8 @@ - (void)viewWillTransitionToSize:(CGSize)size
 
   [coordinator animateAlongsideTransition:
       ^(__unused id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/BottomSheet/src/MDCBottomSheetPresentationController.m]: Done

File: components/BottomSheet/src/MDCBottomSheetPresentationController.m [hunk 4]
```diff
@@ -150,31 +153,32 @@ - (void)dismissalTransitionDidEnd:(BOOL)completed {
...
        withTransitionCoordinator:(id <UIViewControllerTransitionCoordinator>)coordinator {
   [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
 
-  [coordinator animateAlongsideTransition:
-      ^(__unused id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
-        self->_sheetView.frame = [self frameOfPresentedViewInContainerView];
-        [self->_sheetView layoutIfNeeded];
+  [coordinator
+      animateAlongsideTransition:^(
+          __unused id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/BottomSheet/src/MDCBottomSheetPresentationController.m]: Done

**Pull request merged**